cmd/openshift-install/agent.go

```diff
@@ -85,7 +85,20 @@ var (
 		},
 	}
 
-	agentTargets = []target{agentConfigTarget, agentManifestsTarget, agentImageTarget}
+	agentUnconfiguredIgnitionTarget = target{
+		name: "Agent unconfigured ignition",
+		command: &cobra.Command{
+			Use:    "unconfigured-ignition",
+			Short:  "Generates an agent ignition that excludes cluster configuration",
+			Args:   cobra.ExactArgs(0),
+			Hidden: true,
+		},
+		assets: []asset.WritableAsset{
+			&image.IgnitionBase{},
```

Comment:
I've tried to manually use the IgnitionBase asset and getting the following error:
```
failed to fetch dependency of "Agent Installer Ignition Base": failed to generate asset "InfraEnv Config": missing configuration or manifest file
```
The error is thrown from InfraEnv [finish](https://github.com/openshift/installer/blob/14f8ef527b8cb1d1556f1ca882a2344641fc5133/pkg/asset/agent/manifests/infraenv.go#L142) func I guess?